includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -258,6 +261,29 @@ public function decrease_usage_count( &$coupon, $used_by = '' ) {
 				$coupon->set_used_by( (array) get_post_meta( $coupon->get_id(), '_used_by' ) );
 			}
 		}
+		return $new_count;
+	}
+
+	/**
+	 * Increase or decrease the usage count for a coupon by 1.
+	 *
+	 * @since 2.7.0
+	 * @param WC_Coupon
+	 * @param string $operation 'increase' or 'decrease'
+	 * @return int New usage count
+	 */
+	private function vary_usage_count_meta( &$coupon, $operation = 'increase' ) {
+		global $wpdb;
+		$id = $coupon->get_id();
+		$operator = ( 'increase' === $operation ) ? '+' : '-';
+		$new_count = ( 'increase' === $operation ) ? ( $coupon->get_usage_count( 'edit' ) + 1 ) : ( $coupon->get_usage_count( 'edit' ) - 1 );
+
+		$updated = $wpdb->query( $wpdb->prepare( "UPDATE $wpdb->postmeta SET meta_value = meta_value {$operator} 1 WHERE meta_key = 'usage_count' AND post_id = %d;", $id ) );
+		if ( ! $updated ) {
+			add_post_meta( $id, 'usage_count', $new_count, true );
```

Comment:
It may be safer to add beforehand so there is no query error if the meta does not exist?